whitepaper/whitepaper.md

```diff
@@ -706,19 +706,25 @@ Once an API call to Hyperledger Cactus framework is requested by a User, Busines
 
-<img src="./architecture-with-plugin-and-routing.png" width="700">
+<img src="./cactus_arch.svg" width="700">
 
 ### 5.2.1 Definition of key components in system architecture
 
 Key components are defined as follows:
-- **Application user**: The entity submits API calls to "Cactus Routing Interface". Note: this component is exist outside of Cactus service system.
+- ~~**Application user**: The entity submits API calls to "Cactus Routing Interface". Note: this component is exist outside of Cactus service system.~~
 - **Business Logic Plugin**: The entity executes business logic and provide integration services that are connected with multiple blockchains. The entity is composed by web application or smart contract on a blockchain. The entity is a single plugin and required for executing Hyperledger Cactus applications.
+- **Command API Server (cmd-api-server)**: The server accepts a request from an End-user Application, and return a response depending on the status of the targeted trade. Trade ID will be assigned when a new trade is accepted.
+- **End-user Application**: The entity submits API calls to request a trade, which invokes a set of transactions on Ledger by the Business Logic Plugin.
+- **Ledger Event Listener**: The standard interface to handle various kinds of events(LedgerEvent) regarding asynchronous Ledger operations. The LedgerEvent will be notified to appropriate business logic to handle it.
 - **Ledger Plugin**: The entity communicates Business Logic Plugin with each ledger.  The entity is composed by a validator and a verifier as follows. The entity(s) is(are) chosen from multiple plugins on configuration.
+- **Service Provider Application**: The entity submits API calls to control the cmd-api-server when it is enabling/disabling Ledger plugins, or shutting down the server. Additional commands may be available on Admin API since **Server controller** is implementation-dependent.
 - **Validator**: The entity monitors transaction records of Ledger operation, and it determines the result(success, failed, timeouted) from the transaction records.
 Validator ensure the determined result with attaching digital signature with "Validator key" which can be verified by "Verifier".
-- **Verifier**: The entity accepts only sucussfully verified operation results by verifying the digital signature of the validator. Note that "Validator" is apart from "Verifier" over a bi-directional channel.
-- **Cactus Routing Interface**: The entity is a routing service between "Business Logic Plugin" and  "Ledger Plugin(s)". The entity is also a routing service between Business Logic Plugin and API calls from "Application user(s)".
-- **Ledger-n**: DLT platforms(e.g. Ethereum, Quorum, Hyperledger Fabric, ...)
+- **Validator Server**: The server accepts a connection from Verifier, and it provides Validator API, which can be used for controlling and monitoring Ledger behind it. The LedgerConnector will be implemented for interacting with the Ledger nodes.
```

Comment:
> Validator API, which can be used for **controlling** and monitoring Ledger behind it

@sfuji822 Just want to converge on the mental models: What does the `controlling` here refer to? When I read validator, I imagined it would mostly just read ledger state and make statements about that rather than actually being in control of the ledger itself.

Reply:
Discussed at maintainer's call 2020-12-08, controlling as in submitting transactions, not as in owning the entire ledger.
@petermetz

Reply:
I meant controlling is issuing transaction regarding the user's account or assets.
I will change the wording with "issuing signed transactions".
Could you accept this change? @petermetz

Reply:
> I meant controlling is issuing transaction regarding the user's account or assets.
> I will change the wording with "issuing signed transactions".
> Could you accept this change? @petermetz

@sfuji822 Yes, sounds good, thank you!